packages/files-ui/src/Components/Modules/FileBrowsers/MoveFileModal.tsx

```diff
@@ -189,29 +203,44 @@ const MoveFileModule = ({ filesToMove, modalOpen, onClose, onCancel, mode }: IMo
       <Grid
         item
         flexDirection="row"
-        justifyContent="flex-end"
+        justifyContent="space-between"
+        alignItems="center"
         className={classes.paddedContainer}
       >
-        <CustomButton
-          onClick={onCancel}
-          size="medium"
-          className={classes.cancelButton}
-          variant={desktop ? "outline" : "gray"}
-          type="button"
-        >
-          <Trans>Cancel</Trans>
-        </CustomButton>
-        <Button
-          variant="primary"
-          size={desktop ? "medium" : "large"}
-          type="submit"
-          className={classes.okButton}
-          loading={movingFile}
-          disabled={!movePath}
-          onClick={onMoveFile}
+        {movePath === currentPath && (
+          <Typography
+            component="p"
+            variant="body1"
+          >
+            <Trans>The files are already in this folder</Trans>
```

Comment:
We can get fancy here with the translation, we need to import `plural` from lingui.
```suggestion
            <Trans>
              {plural(filesToMove.length, {
                one: `The file is already in this folder`,
                other: `The files are already in this folder`
              })}
            </Trans>
```